lib/games/game.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../item'
```

Comment:
I'd also suggest you to change this path to => './.././../item' I know it's long and weird but I think it's the correct path 😆